### DISCUSSION TITLE
Request for thumbs up/down icon

### DISCUSSION THREAD

**Discussion by slouie2** [twilio-labs/paste#1485] [Request: Icon] [answered]
Hi Paste team, 

My team and I are building out a feature where we would like the customer to give us positive/negative feedback via thumbs up/down icon (currently represented with placeholders ✔️ and **Ⅹ.** in the image below).

![Screen Shot 2021-05-24 at 5 34 12 PM](https://user-images.githubusercontent.com/74426389/119415146-a6984980-bcb6-11eb-8b48-3234fb00bd61.png)

I'd like to put in a request for a thumbs up/down icon. Something similar to this

![image](https://user-images.githubusercontent.com/74426389/119704280-132d5880-be1d-11eb-974a-7667583778bb.png)
![image](https://user-images.githubusercontent.com/74426389/119704320-22aca180-be1d-11eb-8fcd-52b79d580de8.png)

**Comment by TheSisb** [comment]
Hi @slouie2,

Makes sense. Added a ticket to our backlog to tackle these. Will report back when it's done.

**Reply by slouie2** [reply]
Thank you

**Comment by kate-porter** [comment]
Hey @slouie2! I'm working on getting these icons created - how do these look? (This is an example of them mocked up inside buttons.)

![image](https://user-images.githubusercontent.com/5374045/120683747-3da19600-c45b-11eb-9e45-30a0e9fea1d8.png)

**Reply by slouie2** [reply]
The thumbs look great!

**Reply by kate-porter** [reply]
Awesome! We'll followup once they've been added to Paste.

**Reply by TheSisb** [reply]
Heads up, this work is in progress and the icons will be published soon. https://issues.corp.twilio.com/browse/DSYS-2823

**Comment by andioneto** [comment, marked as answer]
@slouie2 Hi there 👋 This was just released and is available from `@twilio-paste/icons` starting with version `5.3.1`. The icon component names are `ThumbsUpIcon` and `ThumbsDownIcon`, and are viewable on the website now. Please take a look and let us know if there is anything else. Thank you!

**Reply by slouie2** [reply]
Looks great. Thanks!
